tests/conftest.py

```diff
@@ -396,3 +402,122 @@ def log_cli_level(pytestconfig):
 
     """
     return pytestconfig.getini('log_cli_level') or 'DEBUG'
+
+
+@pytest.fixture(scope="session")
+def run_io_in_background(request):
+    """
+    Run IO during the test execution
+    """
+    if config.RUN['cli_params'].get('io_in_bg'):
+        log.info(f"Tests will be running while IO is in the background")
+        if config.RUN['google_api_secret']:
+            g_sheet = GoogleSpreadSheetAPI("IO BG results", 0)
+        else:
+            log.warning(
+                "Google API secret was not found. IO won't be reported to "
+                "a Google spreadsheet"
+            )
```

Comment:
It looks like `g_sheet` object won't exist it such case, but later you check it's value. Maybe just setting it to None will be enough here.

Reply:
done